src/SFA.DAS.CaptureTheFlag.Web/Controllers/ForumController.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DAS_Capture_The_Flag.Models.Forum;
+using DAS_Capture_The_Flag.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+
+namespace DAS_Capture_The_Flag.Controllers
+{
+    // Inject an instance of our ForumService from forum layer into the constructor
+    public class ForumController : Controller
+    {
+        private readonly IForumService _forumService;
+        private readonly IPostService _postService;
```

Comment:
Good stuff- Dependency Injection of required components and separating out logic into individual services is great from a Single Responsibility perspective

Also marking the fields as `private readonly` makes the class immutable (the fields can't be changed after the class has been constructed) which means that state is more easily controlled